### DISCUSSION TITLE
Anyone use Borg & Vorta UI on the Windows 11 Linux Subsystem?

### DISCUSSION THREAD

**Discussion by LayersOfAbstraction** [borgbackup/borg#8501] [Q&A]
Hi just want to ask if anyone has gotten Borg working on the Windows 11 Linux Subsystem to back up the entire Windows 11 onto Borg image? 

My computer has two HDDs on a Windows 11 Dynamic Disk. So they sort of load balance each other. Windows 11 is on my SSD. 

Was planning to use Borg and the Vorta UI there on a Ubuntu distro (in WSL) so I could use my Yubikey to encrypt and decrypt my backup external HDDs. 

It would be cool because then me and my friend could exchange HDDs every month and need password as well as Yubikey to crack them.

**Comment by ThomasWaldmann** [comment]
I think getting borg to work under WSL should be relatively simple, but I have no idea about vorta.

**Comment by EnigmaSolved** [comment]
I am curious if you ended up doing this (using Borg and Vorta UI in WSL), and if so, how is working for you?

**Reply by LayersOfAbstraction** [reply]
Sorry, I have moved away from WSL in favor of a Ubuntu Server computer as I deal with multiple devices at home. Feelings are mixed on whether to use Borg which offers encrypted backups or urBackup which does disk cloning too but does not encrypt the backup.

**Reply by EnigmaSolved** [reply]
Appreciate the reply! :)
